addons/Media_Recorder/src_webpack/MediaRecorder.jsm

```diff
@@ -216,7 +225,20 @@ export class MediaRecorder {
     reset() {
         this.deactivate();
         if (this.model.isResetRemovesRecording) {
-            this.resetRecording();
+            if (this.isRecording) {
+                this.resetRecording();
+            } else {
+                this.resetRecording();
+                if (this.model.extendedMode) {
+                    this.setEMDefaultStateView();
+                }
+                this.progressBar.setProgress(0.0);
+                this.keyboardControllerObject.setElements(this._getElementsForExtendedKeyboardNavigation());
```

Comment:
Dopiero sprawdzając Twoją PR ujrzałem błąd, który zrobiłem implementując kod z _getElementsForExtendedKeyboardNavigation. Mianowicie tylko, gdy jest this.model.extendedMod to wtedy powinny być pobierane elementy za pomocą _getElementsForExtendedKeyboardNavigation, w innym przypadku: _getElementsForDefaultKeyboardNavigation. 

LSS: Moim zdaniem nie musisz tego poprawiać - przykuło to moją uwagę i dlatego o tym piszę. Jest to pozostałość, która nie psuje działania addonu a jedynie utrudnia czytanie kodu + nie jest to powiązane z Twoim zadaniem